search/search-client-elasticsearch/src/main/java/io/camunda/search/es/clients/ElasticsearchSessionDocumentClient.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.es.clients;
+
+import co.elastic.clients.elasticsearch.core.SearchRequest;
+import co.elastic.clients.elasticsearch.core.SearchRequest.Builder;
+import io.camunda.search.security.SessionDocumentStorageClient;
+import java.io.IOException;
+import java.util.Map;
+import java.util.function.Consumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ElasticsearchSessionDocumentClient implements SessionDocumentStorageClient {
+
+  private static final Logger LOGGER =
+      LoggerFactory.getLogger(ElasticsearchSessionDocumentClient.class);
+  private final RetryElasticsearchClient client;
+  private final String indexName = String.format("%s-%s-%s_", "camunda", "web-session", "1.1.0");
```

Comment:
❌ We must support with 8.7 one global prefix https://github.com/camunda/camunda/issues/23807

Reply:
Good point, I have renamed the index to start with identity as I didn't want to add a new component.